Update current default option when creating a storage account (LRS in…

laraaleite opened the pull request:
…stead of RA-GRS)

New: Locally-redundant storage (LRS) is the default option when you create a storage account.
Read-access geo-redundant storage (RA-GRS) is the default option when you create a storage account.

azurecla commented:
Hi __@msolhab__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mohabib). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

laraaleite committed:
Update current default option when creating a storage account (LRS instead of RA-GRS)

New: Locally-redundant storage (LRS) is the default option when you create a storage account.
Read-access geo-redundant storage (RA-GRS) is the default option when you create a storage account.
